### DISCUSSION TITLE
How to handle moving savings from Off budget to On budget account

### DISCUSSION THREAD

**Discussion by pepesenaris** [actualbudget/releases#191] [Q&A]
Let's say you have an off budget Savings account that you regularly move money towards.

How do handle the case where you have a big purchase and need to take money out of savings?

Scenario:
On budget  checking account A
Off budget savings account B. 
Savings category that every month is used to moved money from A to B, slowly building up savings while making it part of the budget. 

- Move 1000 from B to A to cover big purchase. (Needs to assign a category here) 
- Pays 1000 from account A (assign Housing category for example, something broke in the house) 

What's the category used to pull money from the account B (off budget) to account A (on budget) 

Is it Saving? 
Is it a category of type Income? 
Is it category that should be used as debt? (In this case debt with yourself) something similar to using a rollover category for old credit card debt where future transfers to the off budget savings account B are classified with this category? 

Another idea? 

Best to all

**Comment by esharden** [comment]
My first question would be, why is your savings off-budget? "Savings" isn't a thing. It's a type of account. If you own a house, really you should have an on-budget category for house repairs or whatever. Why is it debt to yourself? If you're going into debt with yourself to pay for an unexpected expense, you need to get more honest with yourself about whether that money was even available for "savings."

That said, there are a few ways to handle bringing off-budget money back on-budget. No real right or wrong way. If you're determined to keep it off-budget, I'd ask what category you're using to move the money off-budget. Let's say you categorize those moves as Future Big Hairy Expenses. I'd bring it back into the budget using the same category. Or you could bring it back in as income.

**Reply by pepesenaris** [reply]
Hi.
The house is just an example. Not really related to that.

I kept the savings off budget because on one hand it's what the example in the docs recommended and on the other is money generally not intended for month to month expense.
The big purchase just happened, life as in Jurassic park, found a way 😉.

To move the money out of the budget I just used a Savings category.
More specifically this is a new budget and most of it just started as initial balance in the off budget account.

**Reply by pepesenaris** [reply]
It's not about going into debt... Maybe it was poorly put on my part. I mean it as "aim to get to a certain savings amount, so using money from there slows down the speed"

**Reply by esharden** [reply]
Oh, I got that. My point is that if you're "aim(ing) to get a certain savings amount" without having other "savings" for the inevitable Shit Happens stuff that life throws at you, you're just fooling yourself. It was never savings.

I'm not picking on you. I'm pointing out a very common fallacy. We have this general, undefined idea of "building savings" and we scrimp and save and put every spare dollar into "savings." And then we act as though those savings are sacrosanct. It's a lie. You need to "save" in a category for house repairs. And a category for car repairs (if you have a car, of course). And a category for medical deductible or MOOP. And a category for tech repairs. And new appliances. And your homeowner's deductible. That's real savings. Then, when you need to spend it, you start the process of saving for that expense again.

The only savings that really belongs off-budget is retirement savings.

**Reply by esharden** [reply]
> Hi. The house is just an example. Not really related to that.
> 
> I kept the savings off budget because on one hand it's what the example in the docs recommended and on the other is money generally not intended for month to month expense. The big purchase just happened, life as in Jurassic park, found a way 😉.
> 
> To move the money out of the budget I just used a Savings category. More specifically this is a new budget and most of it just started as initial balance in the off budget account.

The technical answer: if you moved money out using the Savings category, that's probably the best way to bring it back into the budget. But you could also bring it back in as income.

My answer as an experienced professional: month-to-month expenses are just one portion of the overall expenses you'll encounter over the course of a year. In addition to monthly expenses, you need to plan for regular non-monthly (quarterly, semi-annual, annual, etc) expenses as well as inevitable expenses that don't happen on any predictable time schedule. The best way to plan for and prepare for these non-monthly expenses is to treat them as if they were a monthly debt -- save a little bit each month until you reach a logical, reasonable target.

No disrespect to the software developer -- I believe Actual is still a one-person project -- but the docs are thin and the advice is questionable. Should you build up some savings? Yes, in the sense that the only way to really get ahead is to have some cash on hand for unexpected stuff. That way you don't keep falling back into the same old hole and keep going around in the same barely solvent circle. But that's Step 0. And hopefully the goal is to progress past that step as soon as possible.

**Reply by Deguello** [reply]
I think the best answer to the question is to bring it back into the budget using the same category you used to take it out. I don't like bringing it back as income because it isn't income. You're just changing the pocket in which you keep the money that was already income.

That said, @esharden is spot on, imo. The savings account is better on budget and the money in it is better assigned to actual categories, no pun intended, and not just sitting there in a big pool, whether it is for month-to-month expenses or not.

For example, my savings account is on budget. I have three category groups I use for the money there that isn't month-to-month. 

Fixed is for non-monthly expenses with a known due date and a relatively known amount. Think Amazon Prime, insurance, Office 365, etc.

Periodic is for non-monthly expenses without a known due date and potentially variable amounts. Think car maintenance, home maintenance, etc. 

Emergency is for insurance deductibles and funding the inevitable HVAC failure or the like.

IMO, if someone is counting something as "savings" but also counting it as available for large expenses, that money is being double-counted, at least implicitly.

**Reply by pepesenaris** [reply]
Thanks for the insights @esharden @Deguello. 
Great point on using retirement accounts as off budget.
Agree it's not as straightforward with savings.

I'll take a look and likely move the account to on budget. Have a great weekend ❤️😁
